### ISSUE TITLE
BatchNorm on Postnet

### ISSUE THREAD

**Issue by CookiePPP** [NVIDIA/tacotron2#421]
https://github.com/NVIDIA/tacotron2/blob/185cd24e046cc1304b4f8e564734d2498c6e2e6f/model.py#L138

I'd like to ask if using BatchNorm on the Postnet output is intend / tested?

https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html
If I understand correctly, the BatchNorm will normalize the values along the channel dimension to have the same mean and std. 
mean=0, std=1 if affine is false, otherwise learned mean/std during training.

Wouldn't this force the Postnet to modify each mel channel equally, even when it would be illogical for the Postnet to do so?
(e.g: maybe the DecoderRNN does a good job with lower frequencies, but has trouble with higher frequencies. This would be a bad situation for the Postnet right? It can't fix one set of mel channels without modifying the other)

I'd like to ask if someone could give their thoughts on this, thanks!
